config_example.sh

```diff
@@ -83,3 +83,31 @@
 # Install v1alpha2 controllers and CRDs
 #
 #export V1ALPHA2_SWITCH=false
+
+# Image OS (can be "Cirros", "Ubuntu", "Centos", overriden by IMAGE_* if set)
+#
+#export IMAGE_OS="Cirros"
+
+# Image for target hosts deployment
+#
+#export IMAGE_NAME="CentOS-7-x86_64-GenericCloud-1907.qcow2"
+
+# Location of the image to download
+#
+#export IMAGE_LOCATION="http://cloud.centos.org/centos/7/images"
```

Comment:
We should probably show the sample IMAGE_NAME/IMAGE_LOCATION as the defaults, e.g now cirros?  That would also be more consistent with the IMAGE_OS above and the IMAGE_USERNAME

Reply:
True, will fix! thank you